### DISCUSSION TITLE
Unable to initiate a transfer money from reveneue account to asset account

### DISCUSSION THREAD

**Discussion by meminens** [firefly-iii/firefly-iii#8143] [Firefly III] [answered]
I have a revenue account for my employer. And I would like to transfer money to my asset account (checking). However, the revenue account is not showing in the drop down menu. Neither source nor destination accounts give me the option to select the revenue account.

![image](https://github.com/firefly-iii/firefly-iii/assets/42714627/8a26acf2-798f-422e-ab5a-d63b0e792557)

Revenue account balance is showing $0 but there is no way to add money to the revenue account. I thought it would show negative numbers as I receive paychecks.

I feel like I don't understand something fundamental. Any help is much appreciated.

Help page tells me to select the revenue account but like I said the revenue account is not listed in the drop down menu for some reason.

https://docs.firefly-iii.org/firefly-iii/financial-concepts/transactions/

`When you wish to create a deposit, select a revenue account first, as the source account.`

**Comment by JC5** [comment, marked as answer]
You can just type the name of the revenue account. It will appear or be automatically created. 

There's no need to put money into a revenue account first.

**Comment by meminens** [comment]
Thank you so much! That works. I somehow though the revenue accounts would show in the dropdown menu which appears after I click on the text field. That dropdown menu shows all of the other accounts (none of them were used before) but not my revenue accounts.

Manually typing in the revenue account immediately showed my existing revenue account. So all is good.
